web/docs/setup/configure.md

```diff
@@ -213,22 +213,22 @@ easier, as plugins can be installed and uninstalled alongside their respective
 configuration.
 
 ```yaml
-# <configdir>/vast/plugin/example.yaml
+# <configdir>/tenzir/plugin/example.yaml
 option: 42
 ```
 
 After initialization with the configuration options, the plugin is fully
-operational and VAST will call its functions at the plugin-specific
+operational and Tenzir will call its functions at the plugin-specific
 customization points.
 
 ### Show plugins
 
 You can get a list of all plugins and their respective version by running
-`vast version`:
+`tenzir version`:
```

Comment:
The output example of this below must be updated, it doesn't reflect the current version operator.

Reply:
Noted down for the second pass.